.storybook/preview.ts

```diff
@@ -24,6 +15,14 @@ const preview: Preview = {
       },
     },
   },
+
+  initialGlobals: {
```

Comment:
```suggestion
  /* This is how you can customise the Vue application used in MDX files. */
  initialGlobals: {
```

I'm not sure if I was very clear. I'd like to keep the comment so that people can fork and edit the addon to suit their needs a little faster.